attestation-service/src/lib.rs

```diff
@@ -327,6 +327,17 @@ fn parse_init_data(data: Option<InitDataInput>) -> Result<(Option<Vec<u8>>, Valu
                             }
                         }
                     }
+
+                    // If there is a policy rego file, replace the full policy file
+                    // with some claims extracted from the file.
+                    if let Some(Value::String(rego_str)) = map.get("policy.rego").cloned() {
+                        if let Some(policy_data) = extract_policy_data(&rego_str) {
+                            // Since we found some policy claims, remove the original
+                            // policy rego from the claims.
+                            map.remove("policy.rego");
+                            map.insert("agent_policy_claims".to_string(), policy_data);
```

Comment:
Note that AA and CDH config still follow the original name `cdh.toml` and `aa.toml`. We'd better use `policy.rego` also here.

If new names are preferred, we should document them somewhere public. Or a user might get confused by the name changes.

Reply:
The cases are a little bit different. For the config files, we are putting the exact same content into the claims, but we are changing the format. For the policy we are only putting part of the policy into the claims.

It might make sense to change `cdh.toml` to `cdh.json`. I guess we could keep all the same names, but I think we probably should not call this `policy.rego` since it is not the policy.